### PR TITLE
infowindow: fix selectall bug

### PR DESCRIPTION
**Pull request by matallo**
fixed a bug where selectall didn't work as intended (unselect all did)

however, infowindow isn't pulling the last changes, despite the new attrs appear in the `onchange` https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/cartodb3/editor.js#L62

found this happening when infowindow isn't visible

CR @alonsogarciapablo 

/cc @javisantana

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js [hunk 1]
```diff
@@ -97,13 +97,15 @@ module.exports = cdb.core.View.extend({
...
 
-    _(names).each(function (f) {
+    _.each(names, function (f) {
       promises.push(self.model._addField(f));
     });
 
     $.when.apply($, promises).then(function () {
       self.model.sortFields();
-      self.model.trigger('change:fields');
+      self.model.trigger('add:fields');
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: Why do we need so many events (`add:fields`, `change:fields`, y `change`)?

Comment (matallo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: my bad for not having investigated further, followed your additions :S https://github.com/CartoDB/cartodb/blame/master/lib/assets/javascripts/cartodb3/data/infowindow-definition-model.js#L115

Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: Perhaps they are all needed... I was just curious...

**Comment by matallo** [comment]
🚀

**Pull request merged**